### ISSUE TITLE
Plus de fonctionnalitees

### ISSUE THREAD

**Issue by quentinglorieux** [LAbigael/Villey#24]
Serait-il possible d'avoir accès au back office de tout le site ?
Nous aimerions faire des modifications sur les contenus des
différentes pages (page d'accueil, présentation, etc.), notamment nous
souhaitons supprimer l'encadré "Editeurs" en page d'accueil des deux
sites et modifier des éléments en page de présentation.

**Comment by LAbigael** [comment]
Non, on fait les modifs nous, eux ils nous donnent une doc pour les faire
